src/main/java/com/emilpiekos/users/User.java

```diff
@@ -0,0 +1,34 @@
+package com.emilpiekos.users;
+
+import org.springframework.web.bind.annotation.RequestParam;
+
+public class User {
+    private final String firstName;
+    private final String lastName;
+    private final int age;
+
+    public User(String firstName,
+                @RequestParam String lastName,
+                @RequestParam int age) {
```

Comment:
Te adnotacje raczej nic nie robią

Reply:
Adnotacje były dodane na etapie zdania, kiedy to użytkownika dodawano z poziomu okienka wprowadzania adresu w przeglądarce. localhost:8080/add?firstName="foo"&lastName="bar"&age=99